input/postgres/statements.go

```diff
@@ -234,6 +236,22 @@ func GetStatements(server *state.Server, logger *util.Logger, db *sql.DB, global
 		if showtext {
 			statementTexts[key] = receivedQuery.String
 		}
+		if queryID.Valid && showtext {
+			server.QueryIdentitiesMutex.Lock()
+			if server.QueryIdentities == nil {
+				server.QueryIdentities = make(state.QueryIdentityMap)
+			}
```

Comment:
Can we do this initialization once somewhere (possibly there is a place where we already initialize other server structs?), instead of rechecking for it in each loop iteration?

Reply:
Yes, I think it makes sense to move it to the `state.Server{...}` struct construction in `main.go`